content/client-lib-development-guide/features.textile

```diff
@@ -682,7 +682,7 @@ h3(#realtime-presence). Presence
 *** @(RTP11c3)@ @connectionId@ filters members by the provided @connectionId@
 * @(RTP12)@ @Presence#history@ function:
 ** @(RTP12a)@ Supports all the same params as REST @Presence#history@
-** @(RTP12b)@ Additionally supports the param @untilAttach@, which if true, will only retrieve messages prior to the moment that the channel was attached or emitted an @UPDATE@ indicating loss of continuity. This bound is specified by passing the querystring param @fromSerial@ with the @Channel#properties.attachSerial@ assigned to the channel in the @ATTACHED@ @ProtocolMessage@ (see "RTL15a":#RTL15a). If the @untilAttach@ param is specified when the channel is not attached, it will result in an error
+** @(RTP12b)@ Does not support the @untilAttach@ param, unlike @Channel#history@
```

Comment:
For me, leaving this spec point here just adds noise. Would it not be cleaner just to delete it entirely?

Reply:
Fair point - my reasoning was that as part of compliance to spec versions that include this change should include actively removing any now-useless code that does what RTP12b used to require, wasn't sure that just dropping the item would carry the same weight.

But perhaps a better solution to that would be removing the spec item and raising github issues in all the libs to make that transition. I'll do that

Reply:
@QuintinWillison this was done